ClosedXML/Excel/Style/Colors/XLColor_Static.cs

```diff
@@ -169,9 +169,20 @@ public static XLColor FromTheme(XLThemeColor themeColor, Double themeTint)
             }
         }
 
-        private static XLColor noColor = new XLColor();
+        internal static bool IsNullOrTransparent(XLColor color)
```

Comment:
Isn't it better to have this a instance method on `XLColor` itself?

Reply:
This would make the usage more complicated. Compare XLColor.IsNullOrTransparent(color) vs (color == null || color.IsTransparent()).

But maybe you're right.

Reply:
No, leave it like it is.